docs/app/services/version/version.service.ts

```diff
@@ -8,18 +8,22 @@ export class VersionService {
     version: BehaviorSubject<Version> = new BehaviorSubject<Version>(Version.Angular);
 
     constructor() {
-        this.toggle(this.toVersion(window.localStorage.getItem('version')));
+        this.setVersion(this.toVersion(window.localStorage.getItem('version')));
     }
 
-    toggle(version: Version): void {
+    setVersion(version: Version): void {
         if (this.version.getValue() !== version) {
             window.localStorage.setItem('version', version.toString());
             this.version.next(version);
         }
     }
 
     toVersion(version: string): Version {
-        return version.toLowerCase() === '0' ? Version.AngularJS : Version.Angular;
+        let value = parseInt(version);
+        if (!value) {
```

Comment:
Change this to `isNaN(value)` in the if statement as parseInt(undefined) will be equal to NaN. Also set the value in this case to the enum rather than a number as we can't be sure that 1 will always represent Angular/AngularJS